homeassistant/components/jellyfin/media_source.py

```diff
@@ -349,20 +360,24 @@ def _get_stream_url(self, media_item: dict[str, Any]) -> str:
         raise BrowseError(f"Unsupported media type {media_type}")
 
 
-def _media_mime_type(media_item: dict[str, Any]) -> str:
+def _media_mime_type(media_item: dict[str, Any]) -> str | None:
```

Comment:
Do we need to handle missing mime type in `_build_track` too?

Reply:
And that's why we review... Yes, of course. Thanks!
I added that now as well.